api/handlers/metadata.py

```diff
@@ -121,6 +124,60 @@ async def get_meta_thumbnail(
     return StreamingResponse(response.iter_bytes(), media_type=response.headers["Content-Type"])
 
 
+@router.get(
+    "/api/datasources/query",
+    status_code=200,
+    response_model=list[Metadata],
+)
+def query_meta(
+    account: Optional[str] = Query(None),
+    container: Optional[str] = Query(None),
```

Comment:
would be possible for account and containers to be str[] instead of str ? or maybe they are str but we do split by comma or something like that.

Reply:
Done